internal/interpreter/ffi.rs

```diff
@@ -158,23 +125,14 @@ pub extern "C" fn slint_interpreter_value_to_bool(val: &ValueOpaque) -> Option<&
 /// array.
 #[no_mangle]
 pub extern "C" fn slint_interpreter_value_to_array(
-    val: &ValueOpaque,
-    out: *mut SharedVector<ValueOpaque>,
+    val: &Box<Value>,
+    out: *mut SharedVector<Box<Value>>,
```

Comment:
> `SharedVector<Box<Value>>`  

this is not ideal because the destructor of SharedVector in C++ will not destroy the Value :-(
But since the only caller actualy re-interprets-cast this to the right thing, this is probably fine.